src/web/mjs/engine/Connector.mjs

```diff
@@ -476,6 +476,28 @@ export default class Connector {
             } );
     }
 
+    fetchGraphQL(request, operationName, query, variables) {
+        if( typeof request === 'string' ) {
+            request = new Request( request, this.requestOptions );
+        }
+        // TODO: check if this will affect (replace) the input parameter?
+        if( request instanceof URL ) {
```

Comment:
This condition is not required since it is assumed at this point that the request is either an `URL` or a `Request`